README.md

```diff
@@ -4,226 +4,35 @@
 [![Coveralls Status](https://coveralls.io/repos/uwescience/sklearn-forest-ci/badge.svg?branch=master&service=github)](https://coveralls.io/r/uwescience/sklearn-forest-ci)
 [![CircleCI Status](https://circleci.com/gh/uwescience/sklearn-forest-ci.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/uwescience/sklearn-forest-ci/tree/master)
 
-**project-template** is a template project for
-[scikit-learn](http://scikit-learn.org/)
-compatible extensions.
+`sklearn-forest-ci` is a Python module for calculating variance and adding
+confidence intervals to scikit-learn random forest regression or classification
+objects. The core functions calculate an in-bag and error bars for
+random forest objects
 
-It aids development of estimators that can be used in scikit-learn pipelines
-and (hyper)parameter search, while facilitating testing (including some API
-compliance), documentation, open source development, packaging, and continuous
-integration.
+Compatible with Python2.7 and Python3.5
 
-## Important Links
-HTML Documentation - http://contrib.scikit-learn.org/project-template/
-
-## Installation and Usage
-The package by itself comes with a single module and an estimator. Before
-installing the module you will need `numpy` and `scipy`.
-To install the module execute:
-```shell
-$ python setup.py install
-```
-or
-```
-pip install sklearn-template
-```
-
-If the installation is successful, and `scikit-learn` is correctly installed,
-you should be able to execute the following in Python:
-```python
->>> from skltemplate import TemplateEstimator
->>> estimator = TemplateEstimator()
->>> estimator.fit(np.arange(10), np.arange(10))
-```
-
-`TemplateEstimator` by itself does nothing useful, but it serves as an example
-of how other Estimators should be written. It also comes with its own unit
-tests under `template/tests` which can be run using `nosetests`.
-
-## Creating your own library
+This module is based on R code from Stefan Wager (see important links below)
+and licensed under the MIT Open Source Initiative
```

Comment:
Could we change this into `licensed under the MIT open source license`. Also, period at the end of the sentence.